src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -2,19 +2,27 @@
 
 import static com.notably.logic.parser.CliSyntax.PREFIX_TITLE;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
+import com.notably.commons.core.path.AbsolutePath;
 import com.notably.logic.commands.OpenCommand;
+import com.notably.logic.correction.AbsolutePathCorrectionEngine;
 import com.notably.logic.parser.exceptions.ParseException;
+import com.notably.model.Model;
 
 /**
  * Represent a Parser for OpenCommand.
  */
 public class OpenCommandParser implements CommandParser<OpenCommand> {
+    private Model notablyModel;
+    private AbsolutePathCorrectionEngine correctionEngine;
 
+    public OpenCommandParser(Model notablyModel) {
+        this.notablyModel = notablyModel;
+        this.correctionEngine = new AbsolutePathCorrectionEngine(notablyModel, 2);
```

Comment:
Let's make this `2` here a constant instead.

Reply:
Noted!